examples/siro_sandbox/sandbox_app.py

```diff
@@ -710,6 +718,12 @@ def _parse_debug_third_person(args, framebuffer_size):
         default=False,
         help="Shows an intro sequence that helps familiarize the user to the scene and task in a HITL context.",
     )
+    parser.add_argument(
```

Comment:
I asked @rutadesai on Slack about if we can just make this the only option for the tutorial (and thus we don't need to complicate the code with multiple tutorial-related command-line args).

Reply:
Done